hexrd/ui/create_polar_mask.py

```diff
@@ -5,15 +5,30 @@
 from hexrd.ui.hexrd_config import HexrdConfig
 
 
-def create_polar_mask(line_data, rsimg, pv):
+def create_polar_mask(line_data):
+    # Calculate current image dimensions
+    eta_min = -180.0
+    eta_max = 180.0
```

Comment:
We  can probably pull these out as constants

Reply:
We have a constant module [here](https://github.com/HEXRD/hexrdgui/blob/master/hexrd/ui/constants.py)

Reply:
Added UI_ETA_MIN_DEGREES and UI_ETA_MAX_DEGREES to constants.py (force push)